tools/patchwork_integration_test.sh

```diff
@@ -0,0 +1,115 @@
+#! /bin/bash
+
+# Author:
+#   Rohit Sarkar <rohitsarkar5398@gmail.com>
+#
+# This work is licensed under the terms of the GNU GPL, version 2.  See
+# the COPYING file in the top-level directory.
+
+set -e
+USAGE=$(cat <<-END
+	$ ./patchwork_integration_test.sh <path-to-public-inbox> <timestamp>
+	Only mails in public inbox since the given <timestamp> will be considered
+	END
+	)
+if [ $# -ne 2 ]
+then
+	echo -e "$USAGE"
+	exit 1
+fi
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+PUBIN_PATH=$1
+SINCE=$2
+MBOX=test-mbox.mbox
+PROJECT=test-patchwork
+INITIAL_IMPORT_SIZE=50 # Percentage of mbox to be imported as a raw mbox
+INITIAL_IMPORT_MBOX=initial.mbox
+
+clean_up() {
+	1>/dev/null 2>&1 docker rm -f pasta-patchwork
+}
+
+trap clean_up EXIT
+
+get_config() {
+	# Get a minimal config for PaStA
+	# For api imports we keep the intial archive field empty
+	if [ $1 = "mbox" ]
+	then
+		initial_archive=$INITIAL_IMPORT_MBOX
+	else
+		initial_archive=""
+	fi
+	config=$(cat <<-END
+		[PaStA]
+		MODE = "mbox"
+		UPSTREAM = "HEAD^..HEAD"
```

Comment:
We need sane default thresholds here, otherwise we'll run into interactive rating. Please set
INTERACTIVE_THRESHOLD = 0.82
AUTOACCEPT_THRESHOLD = 0.82
DIFF_LINES_RATIO = 0.7

HEADING_THRESHOLD = 0.6
FILENAME_THRESHOLD = 0.95